Code/Tools/TestImpactFramework/Runtime/Code/Include/TestImpactFramework/TestImpactClientSequenceReport.h

```diff
@@ -155,6 +155,28 @@ namespace TestImpact
             {
             }
 
+            SequenceReportBase(SequenceReportBase&& report)
+                : m_maxConcurrency(AZStd::move(report.m_maxConcurrency))
+                , m_testTargetTimeout(AZStd::move(report.m_testTargetTimeout))
+                , m_globalTimeout(AZStd::move(report.m_globalTimeout))
+                , m_policyState(AZStd::move(report.m_policyState))
+                , m_suite(AZStd::move(report.m_suite))
+                , m_selectedTestRuns(AZStd::move(report.m_selectedTestRuns))
+                , m_selectedTestRunReport(AZStd::move(report.m_selectedTestRunReport))
+            {
+            }
+
+            SequenceReportBase(const SequenceReportBase& report)
+                : m_maxConcurrency(report.m_maxConcurrency)
+                , m_testTargetTimeout(report.m_testTargetTimeout)
+                , m_globalTimeout(report.m_globalTimeout)
+                , m_policyState(report.m_policyState)
+                , m_suite(report.m_suite)
+                , m_selectedTestRuns(report.m_selectedTestRuns)
+                , m_selectedTestRunReport(report.m_selectedTestRunReport)
+            {
+            }
```

Comment:
minor: This can benefit from using delegate constructor (https://en.cppreference.com/w/cpp/language/constructor#Delegating_constructor) and keep only one constructor doing the assignments.

Reply:
Cool, I'll check this out 👍

Reply:
fixed 👍